minio/api.py

```diff
@@ -1293,6 +1296,7 @@ def presigned_url(self, method,
         return presign_v4(method, url,
                           self._access_key,
                           self._secret_key,
```

Comment:
Same as above, Shall we add session_token in :params ?

Reply:
are you asking for documentation?

Reply:
yes